File: javascript/2486.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let number = lines.shift();
let sum = 0;

while(number != 0){
  for(let i = 0; i < number; i++){
    let [a, b] = lines.shift().trim().split(" ");
    switch(b){
      case "suco":
        sum += parseInt(a) * 120;
        break;
      case "morango":
        sum += parseInt(a) * 85;
        break;
      case "mamao":
        sum += parseInt(a) * 85;
        break;
      case "goiaba":
        sum += parseInt(a) * 70;
        break;
      case "manga":
        sum += parseInt(a) * 56;
        break;
      case "laranja":
        sum += parseInt(a) * 50;
        break;
      case "brocolis":
        sum += parseInt(a) * 34;
        break;
    }
  }
  if(sum >= 110 && sum <= 130){
    console.log(`${sum} mg`);
  }
  else if(sum > 130){
    console.log(`Menos ${(130 - sum) * -1} mg`)
  }
  else if(sum < 110){
    console.log(`Mais ${110 - sum} mg`)
  }

  sum = 0;
  number = lines.shift();
}

File: java/2486.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
import java.io.PrintWriter;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        PrintWriter out = new PrintWriter(System.out);
        
        String line;
        while ((line = br.readLine()) != null && !line.isEmpty()) {
            int number = Integer.parseInt(line.trim());
            
            if (number == 0) {
                break;
            }
            
            int sum = 0;
            
            for (int i = 0; i < number; i++) {
                String[] parts = br.readLine().trim().split(" ");
                int quantidade = Integer.parseInt(parts[0]);
                String fruta = parts[1];
                
                switch (fruta) {
                    case "suco": sum += quantidade * 120; break;
                    case "morango": sum += quantidade * 85; break;
                    case "mamao": sum += quantidade * 85; break;
                    case "goiaba": sum += quantidade * 70; break;
                    case "manga": sum += quantidade * 56; break;
                    case "laranja": sum += quantidade * 50; break;
                    case "brocolis": sum += quantidade * 34; break;
                }
            }
            
            if (sum >= 110 && sum <= 130) {
                out.println(sum + " mg");
            } else if (sum > 130) {
                out.println("Menos " + (sum - 130) + " mg");
            } else {
                out.println("Mais " + (110 - sum) + " mg");
            }
        }
        
        out.close();
        br.close();
    }
}
